
syntax = "proto3";

message Features {
    int32 id = 1;
    float sensor1 = 2 ;
    float sensor2 = 3 ;
    float sensor3 = 4 ;
    float sensor4 = 5 ;
}
message Empty{}

service StreamDataBroker{
    rpc StreamDataBroker( Empty) returns(stream Features);
}